specs/_features/maxeb_increase/capella.py

```diff
@@ -1627,6 +1652,43 @@ def process_pending_balance_deposits(state: BeaconState) -> None:
     state.pending_balance_deposits = state.pending_balance_deposits[next_pending_deposit_index:]
 
 
+def apply_pending_consolidation(state: BeaconState, pending_consolidation: PendingConsolidation) -> None:
+    target_validator = state[consolidation.target_index]
+    source_validator = state[consolidation.source_index]
+
+    # Ensure the source and the target exit statuses have not changed
+    if source_validator.exit_epoch != FAR_FUTURE_EPOCH:
+        return
+    if target_validator.exit_epoch != FAR_FUTURE_EPOCH:
+        return
+
+    # Move active balance
+    active_balance_ceil = MIN_ACTIVATION_BALANCE if has_eth1_withdrawal_credential(source_validator) else MAX_EFFECTIVE_BALANCE
+    active_balance = min(state.balances[consolidation.source_index], active_balance_ceil)
+    state.balances[consolidation.target_index] += active_balance
+    state.balances[consolidation.source_index] = state.balances[consolidation.source_index] - active_balance
+
+    # Change the status of the source
+    source_validator.consolidated_to = consolidation.target_index
+    # Balance is not exiting the active set, do not apply churn
+    source_validator.exit_epoch = get_current_epoch(state)
```

Comment:
```suggestion
    source_validator.exit_epoch = compute_activation_exit_epoch(get_current_epoch(state))
```

Reply:
We can’t keep the source in the active state anymore. If the source’s balance is greater than `MIN_ACTIVATION_BALANCE + EJECTION_BALANCE` then the excess balance will become an active voting balance.

Reply:
Setting `exit_epoch` to the current epoch doesn’t respect the `MAX_SEED_LOKAHEAD` but the protocol doesn’t rely on this parameter anyway.

Reply:
Resolved by setting the consolidation epoch to `compute_activation_exit_epoch(get_current_epoch(state))`, [here](https://github.com/michaelneuder/consensus-specs/pull/12/files#diff-1cdf2318cbed64bdc20bd25631130be090a72ca7ff4f05bbc74c0c63a634e0f1R2080)